packages/test/local-server-tests/src/test/audience.spec.ts

```diff
@@ -5,7 +5,6 @@
 
 import { strict as assert } from "assert";
 
-import { ContainerRuntimeFactoryWithDefaultDataStore } from "@fluidframework/aqueduct/internal";
```

Comment:
I think it's fine for this to continue importing from aqueduct?  Like it probably shouldn't be using ContainerRuntimeFactoryWithDefaultDataStore at all, but if so then aqueduct seems like the better import source.